### PR TITLE
BUILD_DOCKERKIT issue for jax ci mainline SWDEV-482397

### PR DESCRIPTION
**Pull request by kiran-thumma**
Added BUILD_DOCKERKIT  as env for dist_wheels and dist_docker. 

This is to fix QA Blocker issue, https://ontrack-internal.amd.com/browse/SWDEV-482397

### CODE REVIEW

**Review by pramenku — APPROVED** [review]

File: build/rocm/ci_build [hunk 1]
```diff
@@ -43,6 +43,9 @@ def dist_wheels(
     # create manylinux image with requested ROCm installed
     image = "jax-manylinux_2_28_x86_64_rocm%s" % rocm_version.replace(".", "")
 
+    env = os.environ.copy()  
+    env['DOCKER_BUILDKIT'] = '1'
```
Comment (mrodden) [inline build/rocm/ci_build:47]: This file is all PEP8 convention so use double quotes for strings.

running the `blacken.sh` script at build/rocm/tools/blacken.sh on the file would fix it as well.

```suggestion
    env = os.environ.copy()  
    env["DOCKER_BUILDKIT"] = "1"
```

File: build/rocm/ci_build [hunk 2]
```diff
@@ -171,6 +174,9 @@ def dist_docker(
         "--tag=%s" % tag,
     ]
 
+    env = os.environ.copy()  
+    env['DOCKER_BUILDKIT'] = '1'
```
Comment (mrodden) [inline build/rocm/ci_build:178]: Same as above
```suggestion
    env["DOCKER_BUILDKIT"] = "1"
```

**Review by mrodden — CHANGES_REQUESTED** [review]

File: build/rocm/ci_build [hunk 3]
```diff
@@ -171,6 +174,9 @@ def dist_docker(
         "--tag=%s" % tag,
     ]
 
+    env = os.environ.copy()  
+    env["DOCKER_BUILDKIT"] = "1"
```
Comment (mrodden) [inline build/rocm/ci_build:178]: Also this isn't being used at the subprocess call below at line 186

**Review by mrodden — CHANGES_REQUESTED** [review]

File: build/rocm/ci_build [hunk 4]
```diff
@@ -177,6 +180,9 @@ def dist_docker(
     # context dir
     cmd.append(".")
 
+    env = os.environ.copy()
```
Comment (Ruturaj4) [inline build/rocm/ci_build:183]: I don't think env is being passed to `subprocess.check_call`

This means env is not being used in the call to `subprocess.check_call`, and which would only use the default environment variables.

**Review by Ruturaj4 — CHANGES_REQUESTED** [review]

**Review by Ruturaj4 — APPROVED** [review]

**Review by mrodden — APPROVED** [review]
Perfect, thank you for fast fixes.

**Pull request merged**